db/schema.rb

```diff
@@ -234,19 +232,6 @@
     t.integer "latest_nomis_booking_id"
   end
 
-  create_table "request_audits", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
Please don't remove tables from schema.rb

Reply:
I obviously need to merge it again.